reporter/build.gradle.kts

```diff
@@ -95,6 +96,7 @@ dependencies {
     implementation("com.squareup.retrofit2:converter-jackson:$retrofitVersion")
     implementation("com.squareup.retrofit2:retrofit:$retrofitVersion")
     implementation("com.vladsch.flexmark:flexmark:$flexmarkVersion")
+    implementation("commons-codec:commons-codec:$commonsCodecVersion")
```

Comment:
Can we use https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html to avoid introducing a new dependency?

Reply:
I asked @mnonnenmacher as well why he did not use java.util - his reponse:

(1) For consistency, because we are already using commons compress to support other compression formats like XZ. 
(2) Commons compress offers more configuration options which we currently don't use, but might want to in future.

Reply:
I believe you're mixing up commons-codec vs. commons-compress here, or? I agree to use later as we already use it, but not for the former.

Reply:
`Base64` does not support streaming, but streaming is crucial here for large projects.

Btw, commons-codec was already on the classpath as transitive dependency of poi-ooxml and cyclonedx-java.

Reply:
> `Base64` does not support streaming, but streaming is crucial here for large projects.

Doesn't [wrap](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Encoder.html#wrap-java.io.OutputStream-) serve that purpose?

> Btw, commons-codec was already on the classpath as transitive dependency of poi-ooxml and cyclonedx-java.

Ok, good point, in that sense the dependency wasn't new. But I still believe it makes sense to minimize our dependencies on external libraries.

Reply:
Done.